docs/worker-node/install-singularity.md

```diff
@@ -211,6 +211,24 @@ UID        PID  PPID  C STIME TTY          TIME CMD
 user         1     0  0 21:34 ?        00:00:00 ps -ef
 ```
 
+Singularity CLI Options
+-----------------------
+
+Singularity has several command line options and for completeness the documentaion is found [here](http://singularity.lbl.gov/). However, following there are some command line options that are common:
```

Comment:
Can you add the singularity documentation as the top bullet in the `References` section? Also, add this link for singularity support/news while you're at it: http://singularity.lbl.gov/support